### ISSUE TITLE
Nothing showing at localhost:9000/~yourappname

### ISSUE THREAD

**Issue by RoskiDeluge** [urbit/create-landscape-app#59]
I'm trying to launch a testapp on a fake ~zod. After cloning CLA, I  `npm install` and `npm run serve` and the console says the app compiles successfully. But when I try to start the app after mounting with the command |start %testapp I get the following error:
`gall: failed to build agent %testapp`

Not sure which step I'm missing, or if I have the incorrect file hierarchy? 

system: 
Ubuntu 20.04.2.0 LTS
LInuxOS
Urbit 1.5

**Comment by matildepark** [comment]
Did you `|commit %home` before trying to start the app?

**Comment by RoskiDeluge** [comment]
Yes, I did. I just `ctrl-D` my ship and tried to relaunch the app with `|start %testapp` and the console says `[already running %home/%testapp]` but when I visit localhost:9000/~testapp nothing's showing and no errors on the CLA console.

**Comment by RoskiDeluge** [comment]
Now I'm wondering if it's my OS that's the problem? I could try on my Mac.

**Comment by matildepark** [comment]
<img width="883" alt="image" src="https://user-images.githubusercontent.com/20846414/120699504-756e0500-c47e-11eb-8dee-7d9d94d932f1.png">

Was able to get this running fine, still. In order what I did was

1. Start the fakezod, `./urbit -F zod` etc.
2. `|mount %` on the fakezod.
3. `npm i` in the create-landscape-app directory
4. `npm start` and typed in my pier as it told me, it compiled the boilerplate fine. Named it `itsanapp`.
5. `npm run serve` in the create-landscape-app directory
6. `|commit %home` on the fakezod, then `|start %itsanapp`
7. I went to `localhost:9000/~itsanapp`.
